backend/app/core/config.py

```
@@ -13,6 +13,50 @@
+if not GEMINI_API_KEY:
+    raise ValueError("GEMINI_API_KEY environment variable is not set.")
+
+# You can swap in another Gemini model name if desired
+GEMINI_MODEL = "gemini-2.0-flash"
+GEMINI_API_URL = f"https://generativelanguage.googleapis.com/v1beta/models/{GEMINI_MODEL}:generateContent?key={GEMINI_API_KEY}"
+
+# Free/alternative LLM settings (Together API has been removed)
+FREE_LLM_OPTIONS = {
+    "huggingface": {
+        "api_url": "https://api-inference.huggingface.co/models/google/flan-t5-large",
+        "api_key_env": "HUGGINGFACE_API_KEY"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)